apps/web/src/components/Follow/FollowList.tsx

```diff
@@ -66,39 +80,39 @@ export default function FollowList({ userRef }: Props) {
   );
 }
 
-const StyledFollowList = styled.div<{ fullscreen: boolean }>`
+const StyledFollowList = styled.div`
   height: 100%;
-  width: ${({ fullscreen }) => (fullscreen ? '100%' : '540px')};
+  width: 100%;
+  padding-top: 24px;
+  border-top: 1px solid ${colors.porcelain};
   display: flex;
   flex-direction: column;
-
-  ${({ fullscreen }) =>
-    fullscreen
-      ? css`
-          width: 100%;
-          padding: ${MODAL_PADDING_THICC_PX}px 8px;
-        `
-      : css`
-          width: 540px;
-        `}
 `;
 
 const StyledHeader = styled(HStack)`
-  padding-bottom: ${MODAL_PADDING_THICC_PX}px;
-`;
-
-const StyledHeaderText = styled.div`
   display: flex;
+  gap: 12px;
 `;
 
-const StyledHeaderTextRight = styled(StyledHeaderText)`
-  justify-content: flex-end;
-`;
+const StyledSpan = styled.span<{ active: boolean }>`
+  font-family: ${BODY_FONT_FAMILY};
+  line-height: 21px;
+  letter-spacing: -0.04em;
+  font-weight: 500;
+  font-size: 18px;
+
+  @media only screen and ${breakpoints.tablet} {
+    font-size: 18px;
```

Comment:
should this be a different font-size? its the same as mobile

Reply:
updated to 16px for mobile and it looks good I think